firefox-ios/Tests/PerformanceTestPlan.xctestplan

```diff
@@ -36,6 +36,7 @@
         "ExperimentIntegrationTests",
         "FakespotTests",
         "FindInPageTests",
+        "FirefoxSuggestTest\/testFirefoxSuggestExists()",
```

Comment:
We would need the test suite here to be greyd out not only the test.. otherwise the function to collect data for performance tests could be broken when seeing another test suite.
<img width="298" alt="Screenshot 2023-11-17 at 11 47 09" src="https://github.com/mozilla-mobile/firefox-ios/assets/1897507/b9646323-d67f-443d-96c3-29bb9d5f7832">